docs/api/api.yml

```diff
@@ -29,15 +29,13 @@ paths:
               schema:
                 type: string
                 description: A plain text, human readable description of the service
-                example: Bag Index Service is running.
-        500:
-          $ref: '#/components/responses/Unavailable'
+                example: EASY Bag Index running v1.2.3
```

Comment:
```suggestion
                example: EASY Bag Index running <version>
```

Reply:
It is an example, so the v1.2.3 seems OK to me.